### PR TITLE
[14.0][ADD] sale_pricelist_item_advanced

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
Browse pricelist items using special menu and views.

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
LGTM!

**Review by Aldeigja — APPROVED** [review]
Code ok.

**Comment by francesco-ooops** [comment]
@OCA/crm-sales-marketing-maintainers good for merge?

File: sale_pricelist_item_menu/views/product_pricelist_item.xml [hunk 1]
```diff
@@ -0,0 +1,236 @@
+<odoo>
+    <record id="product_pricelist_item_form_view_for_action" model="ir.ui.view">
```
Comment (simahawk) [inline sale_pricelist_item_menu/views/product_pricelist_item.xml]:
```suggestion
    <record id="product_pricelist_item_form_view" model="ir.ui.view">
```

File: sale_pricelist_item_menu/views/product_pricelist_item.xml [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+        <field name="name">product.pricelist.item.form</field>
+        <field name="model">product.pricelist.item</field>
+        <field name="arch" type="xml">
+            <form string="Pricelist Items">
+                <sheet>
+                    <h1>
+                        <field name="name" />
+                    </h1>
+                    <group>
```
Comment (simahawk) [inline sale_pricelist_item_menu/views/product_pricelist_item.xml]: can you name outer groups too?

File: sale_pricelist_item_menu/views/product_pricelist_item.xml [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+                <field name="product_id" />
+                <field name="base_pricelist_id" />
+            </field>
+        </field>
+    </record>
+
+    <record id="product_pricelist_item_revision_tree_view" model="ir.ui.view">
+        <field name="model">product.pricelist.item</field>
+        <field name="inherit_id" ref="product.product_pricelist_item_tree_view" />
+        <field name="mode">extension</field>
```
Comment (simahawk) [inline sale_pricelist_item_menu/views/product_pricelist_item.xml]: this mode is the default, you can drop it

**Review by simahawk — CHANGES_REQUESTED** [review]
Hmm I'm not sure if the name is fully appropriate.
Here I don't see only a menu item, I see a whole alternative management of pricelist items. Isn't it?
What about something like `sale_pricelist_item_management` or `sale_pricelist_item_advanced`... or name yours ;)

**Comment by ilyasProgrammer** [comment]
@simahawk Thanks for review. Updated as you suggested.

**Review by simahawk — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@simahawk can you try merging again?

**Comment by francesco-ooops** [comment]
@pedrobaeza could we?

**Comment by pedrobaeza** [comment]
Let's see, but if nothing has been done for changing the failed merge...

/ocabot merge nobump

**Pull request merged**